lib/fenrir_view/property_types.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
```

Comment:
A lot more readable 👍  Makes you think how good employing something like strict prop types with react would be. Get a lot of this stuff for free.